DependencyInjection/SensioFrameworkExtraExtension.php

```diff
@@ -32,6 +32,10 @@ public function load(array $configs, ContainerBuilder $container)
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
         $loader->load('services.xml');
 
+        if (class_exists('Symfony\Bridge\PsrHttpMessage\HttpFoundationFactoryInterface')) {
+            $loader->load('psr7.xml');
+        }
```

Comment:
What if I add the bridge and don't type-hint the controller (i.e. I'm using the bridge for another purpose, like middleware). In this case, everything is ok - but it's impossible for me to disable these extra listeners that I'm not using.